group_project_v2/group_project.py

```diff
@@ -36,6 +39,8 @@ class GroupProjectXBlock(CommonMixinCollection, DashboardXBlockMixin, DashboardR
     )
 
     CATEGORY = "gp-v2-project"
+    REPORT_FILENAME = "group_project_{group_project_name}_stage_{stage_name}_incomplete_report.csv"
```

Comment:
@e-kolpakov What do you think about adding a timestamp to the filename to make it more convenient to save several exports done at different times throughout the course?